MIP/mip-18/README.md

```diff
@@ -0,0 +1,96 @@
+# MIP-18: Stage 0 Upgradeability and Multisigs
+- **Description**: Proposal to establish two multisigs, one for engineers and another for the foundation, to facilitate the deployment and upgrade of the Movement Token (MOVEToken) and other upgradeable contracts.
+- **Authors**: [Primata](mailto:primata@movementlabs.xyz)
+- **Desiderata**:
+
+## Abstract
+
+MIP-18 advocates for the creation of two multisigs: one managed by Core Contributors engineers to propose contract upgrades and another managed by the Movement Foundation to execute them. This proposal enables testing of multisig services while ensuring secure and transparent upgrade processes for the Movement Token. The multisigs will be deployed on Ethereum, Sepolia, and Holesky (if available) networks.
+
+## Motivation
+
+There is an immediate need to deploy the Movement Token (MOVEToken) using a robust, secure, and auditable upgrade mechanism. By creating two multisigs—one for proposal and one for execution—we ensure a transparent and decentralized governance process over upgrades to critical contracts.
+
+The first multisig allows Core Contributors engineers to propose upgrades via Safe’s multisig service. A remote key will automate the population of upgrade proposals. The second multisig, held by the Movement Foundation, will be responsible for executing the scheduled upgrades after a time delay, ensuring community trust and review.
+
+Also, while developing an independent multisig, handling signature showed to be an issue that might be indicative of better letting signature to be handled by safe.global.
+
+## Specification
+
+### Multisig Setup
+
+1. **Core Contributors Multisig** (4/5 signatures required):  
+   - Role: Proposing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: Andy Bell (0x49F86Aee2C2187870ece0e64570D0048EaF4C751), Richard Melkonian (0xaFf3deeb13bD2B480751189808C16e9809EeBcce), Liam Monninger (0x12Cbb2C9F072E955b6B95ad46213aAa984A4434D), Primata (0xB2105464215716e1445367BEA5668F581eF7d063) (Engineers involved with Token), and a remote key for automated transaction proposals via SafeKit API's [proposeTransaction](https://docs.safe.global/sdk/api-kit#propose-a-transaction-to-the-service).
+   - Justification: We leave some wiggle room incase one of the engineers is away. We still need 3/4 engineers to sign.
+
+2. **Movement Foundation Multisig** (3/5 signatures required, triggerable after 2-day timelock):  
+   - Role: Executing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: TBD.
+
+Both multisigs are part of a timelock mechanism to enforce a 2-day waiting period between the proposal and execution of upgrades. The engineers' multisig schedules the upgrade, while the foundation multisig executes it after the timelock expires.
+
+### Workflow
+
+![Workflow](workflow.png)
```

Comment:
Are the "monkey" and "human" roles acting on the Timelock Controller and Deployer Safe some well-known services from Safe that we can rely on?

Reply:
> Are the "monkey" and "human" roles acting on the Timelock Controller and Deployer Safe some well-known services from Safe that we can rely on?

They are not acting on the Timelock, they only served as Externally Owned Accounts that deployed the initial contracts and facilitated the overall token deployment.

Reply:
Perhaps the diagram should reflect that these accounts can not affect further upgrade process. Is it common understanding that Deploy is a one-off action and the timelock contract is meant to be immutable afterwards?